src/routes/safe/components/Apps/components/AppFrame.tsx

```diff
@@ -85,6 +85,22 @@ const safeAppWeb3Provider = new Web3.providers.HttpProvider(getSafeAppsRpcServic
 const URL_NOT_PROVIDED_ERROR = 'App url No provided or it is invalid.'
 const APP_LOAD_ERROR = 'There was an error loading the Safe App. There might be a problem with the App provider.'
 
+// Some apps still need chain name, as they didn't update to chainId based SDK versions
+// With naming changing in the config service some names aren't the expected ones
+// Ex: Ethereum -> MAINNET, Gnosis Chain -> XDAI
+const getLegacyChainName = (chainName: string, chainId: string) => {
+  let network = chainName
```

Comment:
If you're returning the legacy names in uppercase, wouldn't it be more consistent to use `chainName.toUpperCase()` here?

Reply:
I was doing this before I realized that we also needed this function where .toLowerCase() is used 😅 and left it like that because it doesn't really matter